installers/npm/package.json

```diff
@@ -2,7 +2,6 @@
   "name": "elm",
   "version": "0.19.1-5",
   "description": "Installer for Elm: just downloads the binary into node_modules",
-  "preferGlobal": true,
```

Comment:
`preferGlobal` has been deprecated for a long time, and is even removed in newer npm versions.